ui/src/lib/helpers.tsx

```diff
@@ -593,3 +593,27 @@ export const toKebabCase = (inputString: string): string => {
         .replace(/\s+/g, ' ') // Condense longer whitespace down to one space
         .replace(/\s/g, '-'); // Replace single spaces with hyphen
 };
+
+// Fetches sitemaps of a category from the sitemap S3 bucket and constructs a sitemap index of them.
+export const getSitemapIndexXML = async (category: 'publications' | 'users'): Promise<string> => {
+    let sitemapS3Keys: string[] = [];
+    try {
+        const sitemapsRequest = await api.get('/sitemaps/paths', undefined);
```

Comment:
Should be able to omit the second parameter, undefined would be default if not set

Reply:
The type signature for api.get required a 2nd param; I've tweaked this so that we don't have to pass a 2nd parameter anymore